common/schemas/utils/data_utils.py

```diff
@@ -506,18 +506,22 @@ def get_args_from_parameter(parameter, param_value_dict):
 
     return args
 
-
 def san_check(val):
     return val if val is not None else ''
 
-
-def get_unqualified_id(qual):
-    return qual.split(".")[-1]
-
-def get_sensitive_fields(component):
+def get_compliant_fields(component, project, can_be_exported=False):
```

Comment:
if "can_be_exported" == False, it means "it cannot be exported" or "we don't need to consider the export value" ? 
it is quite confused here.  Maybe we can remove the checking of it .  Remove it as a parameter.  We are working on the field name, not the field value.